drivers/sensor/apds9960/apds9960.c

```diff
@@ -478,6 +478,24 @@ static const struct apds9960_config apds9960_config = {
 	.i2c_address = DT_AVAGO_APDS9960_0_BASE_ADDRESS,
 	.gpio_name = DT_AVAGO_APDS9960_0_INT_GPIOS_CONTROLLER,
 	.gpio_pin = DT_AVAGO_APDS9960_0_INT_GPIOS_PIN,
+#if CONFIG_APDS9960_PGAIN_8X
+	.pgain = APDS9960_PGAIN_8X,
+#elif CONFIG_APDS9960_PGAIN_4X
+	.pgain = APDS9960_PGAIN_4X,
+#elif CONFIG_APDS9960_PGAIN_2X
+	.pgain = APDS9960_PGAIN_4X,
```

Comment:
Typo? Looks like this should be 2X instead of 4X

Reply:
yes, will fix, thanks